examples/local/fivenodenetwork/main.go

```diff
@@ -99,3 +118,23 @@ func run(log logging.Logger, binaryPath string) error {
 	<-closedOnShutdownCh
 	return nil
 }
+
+var colors = []*color.Color{
+	color.New(color.FgGreen),
+	color.New(color.FgYellow),
+	color.New(color.FgBlue),
+	color.New(color.FgMagenta),
+	color.New(color.FgCyan),
+}
+
+type writer struct {
+	colIdx int
```

Comment:
Should we have `col *color.Color` here instead so that we don't need to index into `colors` each time?